src/tools/regex-tester/regex-tester.service.test.ts

```diff
@@ -92,6 +92,51 @@ const regexesData = [
       },
     ],
   },
+  {
+    regex: '\\s([^\\s\\[]+)(?:\\[(\\d+)\\])?:\\s',
```

Comment:
Use String.raw to make this easier to check?

Reply:
Hi, yes, may be clearer